docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3.5'
```

Comment:
Does this really need to be this high version since it also limits version of docker engine it is usable on?